lib/cadet/accounts/notification.ex

```diff
@@ -61,8 +67,10 @@ defmodule Cadet.Accounts.Notification do
     IO.puts("Fetch called")
     IO.inspect(user)
 
-    # Test
-    {:ok, []}
+    Cadet.Accounts.Notification
+    |> where(user_id: ^user.id)
+    |> Repo.all()
+    |> (fn array -> {:ok, array} end).()
```

Comment:
https://github.com/geshuming/cadet/blob/ab3a3c82407717cd3f034849b6edf7319532010b/lib/cadet/accounts/notification.ex#L70-L73

This can be simplified into this

```suggestion
    notifications =
      Cadet.Accounts.Notification
      |> where(user_id: ^user.id)
      |> Repo.all()

    {:ok, notifications}
```